package.json

```diff
@@ -71,7 +71,8 @@
     "typescript": "^4.0.0"
   },
   "dependencies": {
-    "aws-sdk": "^2.553.0",
+    "@aws-sdk/lib-dynamodb": "^3.549.0",
+    "aws-sdk-client-mock": "^4.0.0",
```

Comment:
really small thing, aws-sdk-client-mock is used in tests only so it can be part of devDependencies